packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -9,4 +9,11 @@ describe('Checkbox', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+  it('renders Checkbox with id correctly', () => {
+    const component = renderer.create(
+      <Checkbox label="Standard checkbox" ariaDescribedBy={'descriptionID'} id="my-checkbox" />
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
can we just test the ID of the node? a whole snapshot is a lot of noise if we ever make changes to checkbox that don't affect the ID at all.